### PR TITLE
Add default handler for AOT

### PR DESCRIPTION
**Pull request by sindhuvahinis**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

**Review by lanking520 — APPROVED** [review]

File: serving/docker/partition/properties_manager.py [hunk 1]
```diff
@@ -134,6 +134,13 @@ def set_and_validate_entry_point(self):
...
                 else:
-                    raise FileNotFoundError(
-                        f"model.py not found in model path {self.properties_dir}"
-                    )
+                    engine = self.properties['engine']
+                    if engine == "Deepspeed":
+                        entry_point = "djl_python.deepspeed"
+                    elif engine == "FasterTransformer":
+                        entry_point = "djl_python.fastertransformer"
+                    else:
```
Comment (frankfliu) [inline serving/docker/partition/properties_manager.py:142]:
```
else if self.properties.contains("model_id"):
    entry_point = "djl_python.huggingface"
```

Comment (frankfliu) [reply serving/docker/partition/properties_manager.py:142]: How can we handle transformers-neuronx case?

Comment (sindhuvahinis) [reply serving/docker/partition/properties_manager.py:142]: Can we do AOT in transformers-neuronx case?

Comment (frankfliu) [reply serving/docker/partition/properties_manager.py:142]: Not today, but sometime next week.
We need have a way to detect the script is running on inf2 docker image and on inf2 machine.

Comment (sindhuvahinis) [reply serving/docker/partition/properties_manager.py:142]: Ah I see, Let me do that in the next PR. Also, no AOT for HF as of now.

**Pull request merged**